packages/rum-recorder/src/types.ts

```diff
@@ -1,21 +1,5 @@
-// Alias EventWithTime to Record, to avoid naming clash between RRWeb events and RUM events
```

Comment:
Will we be able to reuse this file from web-ui instead of the duplicate type file?

Reply:
Not just yet, we'll have to figure out how to do so.